lib/cdo/script_constants.rb

```diff
@@ -268,17 +274,11 @@ module ScriptConstants
     DEEPDIVE_DEBUGGING = 'deepdive-debugging'.freeze,
     FREQUENCY_ANALYSIS = 'frequency_analysis'.freeze,
     GAMELAB = 'gamelab'.freeze,
-    HELLO_WORLD_FOOD = 'hello-world-food'.freeze,
```

Comment:
I'm not sure if we want to remove these from the i18n list?

Reply:
This is fine because all [hoc scripts are translated](https://github.com/code-dot-org/code-dot-org/blob/659876989008c7e2548933b0669fec5c84d877d4/lib/cdo/script_constants.rb#L459)

Reply:
Great! Thank you Bethany.

Reply:
ooo but if we keep these ones around for translation we probably do want to keep them them (sorry for the turn around, I'm having trouble keeping all of this straight). But the *-2021 scripts don't need to be added here

Reply:
Oh ya good point! Need these and then to add the -2021 ones above. I will update this PR once everything is copied